examples/functions/timetable/Netex_52_planned_delay.xml

```diff
@@ -0,0 +1,214 @@
+								<Longitude>53.0000</Longitude>
+								<Latitude>0.1000</Latitude>
+							</Location>
+							<TimingPointStatus>timingPoint</TimingPointStatus>
+							<ShortName>Alpha</ShortName>
+							<PublicCode>ALPH</PublicCode>
+							<StopType>onstreetBus</StopType>
+							<VehicleModes>bus</VehicleModes>
+						</ScheduledStopPoint>
+						<ScheduledStopPoint version="any" id="mybus:SSP_077">
+							<Name>Charley Crescent</Name>
+							<Location>
+								<Longitude>53.3000</Longitude>
+								<Latitude>0.3000</Latitude>
+							</Location>
+							<TimingPointStatus>timingPoint</TimingPointStatus>
+							<ShortName>Charley</ShortName>
+							<PublicCode>CHAS</PublicCode>
+							<StopType>onstreetBus</StopType>
+							<VehicleModes>bus</VehicleModes>
+						</ScheduledStopPoint>
+					</scheduledStopPoints>
+					<notices>
+						<Notice id="delay_notice" version="any">
+							<Text>During July the bus is delayed by 5 minutes due to construction work.</Text>
+						</Notice>
+					</notices>
+				</ServiceFrame>
+				<!-- ======TIMETABLE ======= -->
+				<TimetableFrame version="1.0" id="ch:tf">
+					<validityConditions>
+						<AvailabilityCondition version="any" id="all_year_no_longer_available">
+							<Description>regular run</Description>
+							<FromDate>2025-01-01T00:00:00Z</FromDate>
+							<ToDate>2025-12-31T00:00:00Z</ToDate>
+							<IsAvailable>false</IsAvailable>
+							<!--making clear that it is no longer available -->
+						</AvailabilityCondition>
+						<AvailabilityCondition version="any" id="only_july">
+							<Description>regular run</Description>
+							<FromDate>2025-07-01T00:00:00Z</FromDate>
+							<ToDate>2025-07-31T23:59:00Z</ToDate>
+						</AvailabilityCondition>
+						<AvailabilityCondition version="any" id="after_july">
+							<Description>regular run</Description>
+							<FromDate>2025-08-01T00:00:00Z</FromDate>
+							<ToDate>2025-12-31T23:59:00Z</ToDate>
+						</AvailabilityCondition>
+						<AvailabilityCondition version="any" id="before_july">
+							<Description>regular run</Description>
+							<FromDate>2025-01-01T00:00:00Z</FromDate>
+							<ToDate>2025-06-30T23:59:00Z</ToDate>
+						</AvailabilityCondition>
+					</validityConditions>
+					<vehicleJourneys>
+						<ServiceJourney version="any" id="444">
+							<validityConditions>
+								<AvailabilityConditionRef ref="all_year_no_longer_available" version="any"/>
+							</validityConditions>
+							<Description>The original journey replaced by 444-1 and 444-2</Description>
+							<ServiceAlteration>replaced</ServiceAlteration>
+							<DepartureTime>14:20:00.0Z</DepartureTime>
+							<LineRef version="any" ref="mybus:LN_24"/>
+							<calls>
+								<Call version="any" id="hde:sj_24o_01_001" order="1">
+									<ScheduledStopPointRef version="any" ref="mybus:SSP_001"/>
+									<Arrival>
+										<ForAlighting>false</ForAlighting>
+									</Arrival>
+									<Departure>
+										<Time>14:20:00.0Z</Time>
```

Comment:
Would you agree that you should skip all the "Z" in Time?

Reply:
remove them? why?

Reply:
Does not make any sense, and your availability conditions cross day light saving.

Reply:
That sounds reasonable.

Reply:
done